frontend/src/entities/dataset/dataset.service.tsx

```diff
@@ -121,13 +123,16 @@ export const fetchPresignedURL = async (s3Key: string) => {
 export const determineScope = (
     type: DatasetType | undefined,
     projectName: string | undefined,
+    groupName: string | undefined,
     username: string
 ): string => {
     switch (type) {
         case DatasetType.GLOBAL:
             return DatasetType.GLOBAL;
         case DatasetType.PROJECT:
             return projectName!;
+        case DatasetType.GROUP:
+            return groupName!;
```

Comment:
This `!` isn't go unnoticed, I just don't have a nicer way to clean that up right now O_o